src/site/articles/style-guide/index.markdown

```diff
@@ -1556,6 +1556,59 @@ followed by more alphanumeric text, the `{}` can and should be omitted.
 {% endprettify %}
 </div>
 
+## Collections
+
+#### PREFER literals over constructors.
+{:.no_toc}
+
+Dart has list and map literals.
+
+<div class="good">
+{% prettify dart %}
+var points = [];
+var addresses = {};
+{% endprettify %}
+</div>
+
+<div class="bad">
+{% prettify dart %}
+var points = new List();
+var addresses = new Map();
+{% endprettify %}
+</div>
```

Comment:
Might be good to mention they can take generic type arguments (`<int>[]`) too if the user wants. Otherwise, they might think they have to use `new List<int>()` to get that.